src/main/java/org/alliancegenome/curation_api/dao/GeneExpressionAnnotationDAO.java

```diff
@@ -0,0 +1,13 @@
+package org.alliancegenome.curation_api.dao;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import org.alliancegenome.curation_api.dao.base.BaseSQLDAO;
+import org.alliancegenome.curation_api.model.entities.GeneExpressionAnnotation;
+
+@ApplicationScoped
+public class GeneExpressionAnnotationDAO  extends BaseSQLDAO<GeneExpressionAnnotation> {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.SingleSpaceSeparatorCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Use a single space to separate non-whitespace characters.